Packs/Microsoft365Defender/IncidentFields/incidentfield-Microsoft_365_Defender_Classification.json

```
@@ -0,0 +1,26 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "microsoft365defenderclassification",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_microsoft365defenderclassification",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "Microsoft 365 Defender Classification",
+    "neverSetAsRequired": false,
+    "ownerOnly": false,
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "shortText",
+    "unmapped": false,
+    "unsearchable": false,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "5.0.0"
```

```suggestion
    "fromVersion": "5.5.0"
```
if the integration is from 5.5.0 we should make all the accompanying entities from that version as well.